.rubocop.yml

```diff
@@ -44,8 +43,6 @@ Lint/ScriptPermission:
   Enabled: false
 Lint/StructNewOverride:
   Enabled: false
-Style/GlobalVars:
-  Enabled: false
```

Comment:
#### Do not modify linter file; undo any changes.

Reply:
Undone